lib/agent0/agent0/hyperdrive/policies/zoo.py

```diff
@@ -3,9 +3,11 @@
 
 from typing import NamedTuple
 
+from .arbitrage import Arbitrage
```

Comment:
something got messed up with the import order. Can you delete the comment on line 8 & resort?

Reply:
I sorted them in order of smarts 😜